### PR TITLE
trinity 2.11.0

### PR DESCRIPTION
**Pull request by SeekingMeaning**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by sjackman** [comment]
```
Errno::ENOENT: No such file or directory @ apply2files - Butterfly/Butterfly/src/src
```
https://github.com/brewsci/homebrew-bio/pull/1090/checks?check_run_id=883368259#step:5:571

**Comment by sjackman** [comment]
On macOS
```
sift_bam_max_cov.cpp:142:10: error: 'string' is not a member of 'std'
```
https://github.com/brewsci/homebrew-bio/pull/1090/checks?check_run_id=883368282#step:4:452

File: Formula/trinity.rb [hunk 1]
```diff
@@ -46,7 +44,14 @@ def install
...
 
-    system "make", "all", "plugins", "test"
+    # args = []
+    # args << "CXXFLAGS=-L#{Formula["libomp"].opt_lib} -lomp" if OS.mac?
+    if OS.mac?
+      ENV.append "CXXFLAGS", "-L#{Formula["libomp"].opt_lib}"
+      ENV.append "CXXFLAGS", "-lomp"
+    end
+
+    system "make", "all", "plugins", "test", *args
```
Comment (sjackman) [inline Formula/trinity.rb:54]:
```suggestion
    system "make", "all", "plugins", "test"
```

Comment (SeekingMeaning) [reply Formula/trinity.rb:54]: Whoops

**Comment by jonchang** [comment]
Looks like this needs autotools

File: Formula/trinity.rb [hunk 2]
```diff
@@ -35,6 +37,23 @@ class Trinity < Formula
   fails_with :clang # needs openmp
 
   def install
+    ENV.cxx11
```
Comment (sjackman) [inline Formula/trinity.rb:40]: I thought this line was a no-op? I may be wrong. I'm probably wrong.

Comment (sjackman) [reply Formula/trinity.rb:40]: https://github.com/Homebrew/brew/blob/1644ec2f50047f026781486f37a9c26c2a64bf90/Library/Homebrew/extend/ENV/super.rb#L271-L274
So it's not a no-op, but I still thought that C++11 was the default standard for GCC 5 and modern Clang.

**Pull request merged**